### ISSUE TITLE
test.py fails on caching duration

### ISSUE THREAD

**Issue by dulfox** [multiprocessio/dsq#80]
I have installed dsq on my manjaro (arch linux like) raspberry pi 4 with this file: [PKGBUILD](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=dsq)

Everything has worked fine until the check part which runs the test.py script.

I have one failure (48 of 49 succeeded) related to this line:
https://github.com/multiprocessio/dsq/blob/8552873f2d87d1d6f9ad1246fa0daac1eb5d833b/scripts/test.py#L303

```
cat taxi.csv | ./dsq --cache -s csv 'SELECT passenger_count, COUNT(*), AVG(total_amount) FROM {} GROUP BY passenger_count ORDER BY COUNT(*) DESC'
```
**FAILURE: completed in 9 seconds. Wanted <5s**

I have bypassed manually this test.

So, my questions are:
why do you want a less than 5 seconds result ?
Why is it a failure ?
How can we compute the "correct time duration" related to **devices** (RAM size, CPU clock, thread number, etc.) ?

**Comment by eatonphil** [comment]
Yeah this is not a great test for exactly the reasons you mention.

You can safely ignore this (and you can script ignoring this with `./scripts/test.py -nf cach`).

Happy to leave this open until I have a better idea on how to test this and time to implement it.
